arp_scripts/submit_smd.sh

```diff
@@ -212,6 +217,8 @@ fi
 SBATCH_ARGS="-p $QUEUE --ntasks-per-node $TASKS_PER_NODE --ntasks $CORES --exclusive -o $LOGFILE"
 MPI_CMD="mpirun -np $CORES python -u ${ABS_PATH}/${PYTHONEXE} $*"
 
+export SIT_PSDM_DATA=/sdf/data/lcls/drpsrcf/ffb
+
```

Comment:
I'm not sure why SIT_PSDM_DATA is fixed here to the FFB on S3DF rather than leaving it where it was before.

Reply:
Looks like a mistake indeed. It should be set on line 192, according to the elog information